### PR TITLE
Update civicrm.module install links

### PR DESCRIPTION
**Pull request by eileenmcnaughton**
@MikeyMJCO probably has a better suggestion.....

### CODE REVIEW

File: civicrm.module [hunk 1]
```diff
@@ -189,7 +189,7 @@ function civicrm_initialize() {
     }
 
     // get ready for problems
-    $docLinkInstall = "http://wiki.civicrm.org/confluence/display/CRMDOC/Drupal+Installation+Guide";
+    $docLinkInstall = "http://www.sorry-no-idea";
```
Comment (homotechsual) [inline civicrm.module]:
```suggestion
    $docLinkInstall = "https://docs.civicrm.org/installation/en/latest/drupal7/";
```

**Review by homotechsual — CHANGES_REQUESTED** [review]

File: civicrm.module [hunk 2]
```diff
@@ -189,7 +189,7 @@ function civicrm_initialize() {
...
     $docLinkTrouble = "http://wiki.civicrm.org/confluence/display/CRMDOC/Installation+and+Configuration+Trouble-shooting";
```
Comment (homotechsual) [inline civicrm.module:193]:
```suggestion
    $docLinkTrouble = "https://docs.civicrm.org/installation/en/latest/general/troubleshooting/";
```

File: civicrm.module [hunk 3]
```diff
@@ -189,7 +189,7 @@ function civicrm_initialize() {
...
     $forumLink      = "http://forum.civicrm.org/index.php/board,6.0.html";
```
Comment (homotechsual) [inline civicrm.module]:
```suggestion
    $forumLink      = "https://civicrm.stackexchange.com/questions/tagged/drupal";
```

**Review by homotechsual — CHANGES_REQUESTED** [review]

**Comment by seamuslee001** [comment]
@eileenmcnaughton are you able to update this PR?

**Comment by eileenmcnaughton** [comment]
#629 closes

**Pull request closed without merging**